profiles/base.yaml

```diff
@@ -186,8 +186,10 @@ dump_item_count: 9
 # as an example, chuno currently runs it at 6 being circle 195
 # defaulting to 25, as 20 for lower skilled barbs was explained to be common
 meditation_pause_timer: 25
-# use the analyze combos to train
+# use the analyze combos to train for the barb - is deprecated!
```

Comment:
💡 This looks like a good candidate for a validator

Reply:
Yeah, adding that now actually